rasa/cli/arguments/interactive.py

```diff
@@ -18,41 +19,53 @@
 )
 
 
-def set_interactive_arguments(parser: argparse.ArgumentParser):
+def set_interactive_arguments(parser: argparse.ArgumentParser) -> None:
```

Comment:
Can we re-use the `set_interactive_core_arguments` function in here? I think `interactive` should have all those in `interactive core` plus a few more

Reply:
No, there are a couple of differences. `interactive core` has a different help text for the `model` parameter and has a `stories` parameter that `interactive` hasn't.

Reply:
Got it, thanks!